polaris.shopify.com/content/content/actionable-language.md

```diff
@@ -586,18 +586,21 @@ Use the adjective “Done” for datepickers.
 
 ---
 
-## OK vs. accept
+## Close vs. accept
 
-Use the adjective “OK” when merchants need to confirm they’ve read something, but aren’t required to legally accept terms of service before continuing. For example, use “OK” when presenting a security notification in a modal or sheet.
+Use the adjective “Close” when merchants need to confirm they’ve read something, but aren’t required to legally accept terms of service before continuing. For example, use “Close” when presenting a security notification in a modal or sheet.
+
+Don't use "OK" as the call to action because it’s an exclamation, not an action. When merchants click the "Close" button, they’re not saying “OK”, they’re doing a specific action.
 
 <!-- dodont -->
 
 #### Do
 
-![modal with okay button spelled with capitalized letter O and K, no punctuation](/images/content/actionable-language/ok@2x.png)
+![modal with Close button](/images/content/actionable-language/ok@2x.png)
```

Comment:
I need to replace this image! ❗

Reply:
Sure I can help! I'll add a commit with the updated image.